app/assets/javascripts/admin/addon/components/admin-config-areas/themes.gjs

```diff
@@ -82,12 +82,24 @@ export default class AdminConfigAreasThemes extends Component {
   }
 
   <template>
+    <DPageSubheader
+      @titleLabel={{i18n
+        "admin.config_areas.themes_and_components.themes.title"
+      }}
+      @descriptionLabel={{i18n
+        "admin.config_areas.themes_and_components.themes.description"
+      }}
+    >
+      <:actions as |actions|>
+        <actions.Primary
+          disabled={{this.loading}}
```

Comment:
```suggestion
```

I don't think there is a `loading` property for this component and since this button is just opening a modal, I don't think there is a need to disable it as well.

I think we should remove the following line as part of this PR too since switching between `Themes` and `Components` is causing a "flash" with the install button.

https://github.com/discourse/discourse/blob/09a457f2ab3e0f3c0e78eb105dcbe0f98afcf85b/app/assets/javascripts/admin/addon/components/admin-config-areas/components.gjs#L175

![Kapture 2025-05-29 at 10 53 55](https://github.com/user-attachments/assets/a02a3115-2548-4f92-bf37-4a26f2558eb3)